python/tests/conftest.py

```diff
@@ -13,11 +13,10 @@
 
 def wait_till_host_is_available(host: str, timeout_sec: int = 30):
     spacing = 2
-    attempts = timeout_sec / spacing
     while True:
         try:
             subprocess.run(["curl", host], timeout=500, check=True)
-        except:
+        except BaseException:
```

Comment:
I think we should not be catching base exception here, but rather `Exception`.

learned the hard way once what it says in pep8 :).

> Direct inheritance from BaseException is reserved for exceptions where catching them is almost always the wrong thing to do.

The case that comes to mind is e.g. keyboard interrupts, which we would catch with the BaseException as well.